app_dart/lib/src/foundation/utils.dart

```diff
@@ -136,6 +138,12 @@ Future<List<Target>> getTargetsToRun(Iterable<Target> targets, List<String?> fil
       }
     }
   }
+
+  log.info('Collected the following targets to run:');
+  for (var element in targetsToRun) {
```

Comment:
```suggestion
  for (final element in targetsToRun) {
```

https://dart-lang.github.io/linter/lints/prefer_final_locals.html